app/services/recaptcha_validator.rb

```diff
@@ -55,6 +61,10 @@ def faraday
     end
   end
 
+  def logged_recaptcha_result(recaptcha_result)
+    recaptcha_result
+  end
```

Comment:
this was added just so the subclass could override it?

How long do we expect both enterprise/non-enterprise to stay around?

If we expect for a long time, my approach would be to see if we can have classes that return consistent structs from both services, and then the validator just has to process that struct

(but you probably knew I'd ask 😜 )

Reply:
That's an interesting idea, I can play around with it.

As far as how long we'd maintain both, I'm not really sure yet. It's true that eventually we'd probably only want the Enterprise version, as it'd be what we'd use in production, and could reasonably support for local development. But there's a lot more friction to getting access to Enterprise vs. generating tokens from the free version of the product, where the latter is a few clicks away for most anyone (even outside contributors, etc).

Reply:
In retrospect, we could probably log the whole response. I think I was initially concerned that the [`event` response property](https://cloud.google.com/recaptcha-enterprise/docs/reference/rest/v1/projects.assessments#Event) included a key which I assumed was the application's secret key, but in fact it's the non-sensitive "site" key.

That being said, I still see some value in normalizing what we're logging, which a struct could help with.

Reply:
Gotcha, yeah, now that we're having this conversation, I'm not totally convinced we want to continue maintaining multiple implementations long-term. I might open a refactor ticket to drop the free validator once we feel solid about the Enterprise implementation.